ansible/roles/openondemand/tasks/vnc_compute.yml

```diff
@@ -48,6 +48,7 @@
   tags: install
   yum:
     name: '@Xfce'
+  when: appliances_mode != 'configure' # dnf group/module installs aren't idempotent so only run during build
```

Comment:
Ok - this task does run during fatimage.

Reply:
yeah, I've tested a remote desktop when it's installed just in fatimage and it seems to work okay